containers/runner/README.md

```diff
@@ -18,7 +18,7 @@ This will download the [automatically built](.github/workflows/container-autoupd
 
 You can also build the container yourself to test modifications to it:
 
-    podman build -t rhinstaller/kstest-runner .
+    podman build -t rhinstaller/kstest-runner containers/runner/
```

Comment:
Please also add --no-cache to avoid using cache and skipping the build. That would be most probably what you really want and avoid unexpected issues.

Reply:
Good point, fixed, thanks.